terraform/modules/s3/s3.tf

```diff
@@ -0,0 +1,71 @@
+# TODO add proper encryption
+resource "aws_s3_bucket" "s3_uploads" {
+  bucket = var.s3_bucket_name
+  # TODO look into better policies later
+  acl = "private"
+
+  # TODO improve this security wise
+  cors_rule {
+    allowed_headers = ["*"]
+    allowed_methods = ["PUT", "POST", "DELETE"]
+    allowed_origins = ["http://localhost:3000", "https://staging.socialdiversity.org"]
```

Comment:
I'm not sure we can add pr deploys here since the pr domains would be dynamic right? And would probably want to be added and removed in the pr lifecycle? I'm actually not sure what to do about this

Reply:
> Access-Control-Allow-Origin Headers can only be a wildcard '*' or an exact url. 

hmm, okay. Let's just keep PR deploys out of this.

Reply:
Oh true the wildcard! I think that’s possible